change/@ni-nimble-components-00ae0774-00b7-4933-94bd-ae498706be78.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I found that using npm 10 with SystemLinkShared results in a lot of warnings being printed to the console.

>(node:43935) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 close listeners added to [TLSSocket]. Use emitter.setMaxListeners() to increase limit

I guess that upgrading Storybook might fix the issue since it uses node-fetch and that reports a similar bug here: https://github.com/node-fetch/node-fetch/issues/1295

I'm going to wait to move this out of draft for now and look at upgrading Storybook first.

Reply:
Other than the warnings, upgrading Node in Skyline seems to cause no problems with Angular pipelines: https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/588986

Reply:
I no longer see those warnings after the storybook update.